src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -70,6 +70,8 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null): void
     public function getListeners($event = null): array
     {
         if (null === $event) {
+            trigger_deprecation('symfony/doctrine-bridge', '6.2', 'Calling "%s()" without an event name is deprecated. Call getAllListeners() instead.', __METHOD__);
```

Comment:
This should be done only if `getAllListeners` exists in the parent class (and so if this usage is deprecated in the parent method as well). There is 2 ways to do that:

- bump the min version to 1.2 (it is currently 1.0)
- detect this (with `method_exists(EventManager::class, 'getAllListeners')`)

Reply:
Good point. Option 1 sounds like the simpler solution.

Reply:
@derrabus Merged up now :)